### DISCUSSION TITLE
Expand tasks w/glob

### DISCUSSION THREAD

**Discussion by jaredpalmer** [vercel/turborepo#12601] [Ideas]
### Describe the feature you'd like to request

While you can run multiple tasks at the same time (i.e. muilti entry task running), you have to type each task out by hand at the moment:

```sh
turbo run lint:eslint lint:prettier
```

That's annoying....

### Describe the solution you'd like

Imagine a pipeline that contains 2 tasks named: `lint:eslint` and `lint:prettier`, you should be able to run

```sh
turbo run lint:*
```

and this should be the equivalent of 

```sh
turbo run lint:eslint lint:prettier
```

### Describe alternatives you've considered

n/a

**Comment by jlarmstrongiv** [comment]
`npm-run-all` has [glob matching](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/npm-run-all.md#glob-like-pattern-matching-for-script-names) and distinguishes between `watch:*` and `watch:**`.  

Personally, I name many scripts like `turbo:taskname:check` and `turbo:taskname:update`, so being able to use globs arbitrarily like `turbo:*:check` would be nice.
